content/blog/2018/06/2018-06-17-running-jenkins-with-java10-11.adoc

```diff
@@ -0,0 +1,125 @@
+the packages are ready for evaluation and exploratory testing.
+
+This article explains how to run Jenkins with Java 10 and 11 using Docker images and WAR files.
+It also lists known issues and provides contributor guidelines.
+
+=== Running in Docker
+
+In order to simplify testing, we have created a new
+link:https://hub.docker.com/r/jenkins/jenkins-experimental/[jenkins/jenkins-experimental]
+repository on DockerHub.
+This repository includes various Jenkins Core images, including Java 10 and Java 11 images.
+We have also set up development branches and continuous delivery flows for Jenkins core,
+so now we can deliver patches for these images without waiting for weekly releases.
+
+You can run image simply as:
+
+```
+docker run -p 8080:8080 -p 50000:50000 jenkins/jenkins-experimental:latest-jdk10
+```
+
+The following tags are available:
+
+* `2.127-jdk10`, `2.127-jdk11` - Weekly releases packaged with Java 10 and 11
+* `latest-jdk10` - Jenkins core build from the link:https://github.com/jenkinsci/jenkins/tree/java10-support[java10-support] branch
+* `latest-jdk11` - Automatic build from the core's link:https://github.com/jenkinsci/jenkins/tree/java11-support[java11-support] branch.
+
+Java 10/11 images are fully compatible with the official
+link:https://github.com/jenkinsci/docker/blob/master/README.md[jenkins/jenkins]
+images in terms of options.
+E.g. you can use `plugins.txt` to install plugins and pass extra options.
+
+=== Running Jenkins without Docker
+
+==== Java 10
+
+1. Download Jenkins WAR for 2.127 or above
+** You can also use experimental branches referenced above
+2. Run WAR with the following command:
+
+```shell
+${JAVA10_HOME}/bin/java --add-modules java.xml.bind -jar jenkins.war \
+    --enable-future-java --httpPort=8080 --prefix=/jenkins
+```
+
+==== Java 11
+
+1. Download Jenkins WAR for 2.127 or above
+** You can also use experimental branches referenced above
+2. Download the following libraries to the same directory as jenkins.war
+** link:http://central.maven.org/maven2/javax/xml/bind/jaxb-api/2.3.0/jaxb-api-2.3.0.jar[jaxb-api-2.3.0.jar] (save as jaxb-api.jar)
+** link:http://central.maven.org/maven2/com/sun/xml/bind/jaxb-core/2.3.0.1/jaxb-core-2.3.0.1.jar[jaxb-core-2.3.0.1.jar] (save as jaxb-core.jar)
+** link:http://central.maven.org/maven2/com/sun/xml/bind/jaxb-impl/2.3.0.1/jaxb-impl-2.3.0.1.jar[jaxb-impl-2.3.0.1.jar] (save as jaxb-impl.jar)
+** https://github.com/javaee/activation/releases/download/JAF-1_2_0/javax.activation.jar[javax.activation v.1.2.0]  (save as javax.activation.jar)
+3. Run the following command:
+
+```shell
+Run Jenkins with ${JAVA11_HOME}/bin/java \
+    -p jaxb-api.jar:javax.activation.jar --add-modules java.xml.bind,java.activation \
+    -cp jaxb-core.jar:jaxb-impl.jar \
+    -jar jenkins.war --enable-future-java --httpPort=8080 --prefix=/jenkins
+```
+
+=== Current state
+
+As of June 17, we have achieved the following state:
```

Comment:
It would make sense to separate into two lists: What works (first four items), and what doesn't (the others).

I read the items on top expecting this to be a list of things that work, and then it took a turn.

Reply:
done